### ISSUE TITLE
FR: Books bound on the right-hand side spread with inverted order

### ISSUE THREAD

**Issue by jeltaqq** [kolyvan/kybook#184]
iOS 12.3
KyBook 3 v0.7.8 | #165

Hello. Books bound on the right-hand side, such as a Japanese document.
It's great if KyBook 3 supports those Books.

![img1_2](https://user-images.githubusercontent.com/25340049/60767679-14a0eb00-a0f6-11e9-9999-5735115c3b05.png)

**Comment by ghost** [comment]
Yes please. I would love to have right to left support for my Manga.
